packages/components/src/components/ScalarModal/ScalarModal.vue

```diff
@@ -119,7 +119,7 @@ export function useModal() {
         </div>
         <DialogDescription
           v-else
-          :class="cx(bodyClass, body({ size, variant }))">
+          :class="cx(variant, body({ size }), bodyClass)">
```

Comment:
Variant still needs to be passed into the `body` function

```suggestion
          :class="cx(body({ size, variant }), bodyClass)">
```

Reply:
this doesn't prevent important usage i'm afraid :(

Reply:
good call updated!